tips/tests/test_rule_engine.py

```diff
@@ -239,3 +108,80 @@ def test_apply_rules_recursing(self):
         rules = [{"type": "ref", "ref_id": "1"}]
         with self.assertRaises(RecursionError):
             apply_rules(self.test_data, rules, compound_rules)
+
+    def test_stadspas(self):
+        fixture = get_fixture()
+        user_data = fixture
+        rules = [
+            {"type": "ref", "ref_id": "1"} # ID 1 is the stadspas rule
+        ]
+        self.assertTrue(apply_rules(self.user_data, rules, compound_rules))
+        # Change birth date so test will assert differently
+        user_data['brp']['persoon']['geboortedatum'] = '2012-01-01T00:00:00Z'
+        self.assertFalse(apply_rules(self.user_data, rules, compound_rules))
+
+    def test_is_18(self):
+        fixture = get_fixture()
+        user_data = fixture
+        rules = [
+            {"type": "ref", "ref_id": "2"} # ID 1 is the stadspas rule
+        ]
+        self.assertTrue(apply_rules(self.user_data, rules, compound_rules))
+        # Change birth date so test will assert differently
+        user_data['brp']['persoon']['geboortedatum'] = '2012-01-01T00:00:00Z'
+        self.assertFalse(apply_rules(self.user_data, rules, compound_rules))
```

Comment:
Hier gebruik je ook weer `user_data` en `self.user_data`, en in alle volgende ook.